src/urbit/app/chess.hoon

```diff
@@ -481,8 +485,11 @@
   |=  =path
   ^-  (quip card _this)
   ?+  path  (on-watch:default path)
-    ::
-    ::  convert incoming challenges to chess-update marks for subscribers
+  ::
+  ::  XX: should we rename this to
+  ::      %challenges-received for consistency's sake?
+  ::
+  ::  convert incoming challenges to chess-update marks for subscribers
```

Comment:
Old indentation was correct (same goes for all changes to comment indentation in this file)